tools/cldr-code/src/main/java/org/unicode/cldr/test/SubmissionLocales.java

```diff
@@ -102,39 +166,39 @@ public final class SubmissionLocales {
      */
     public static boolean allowEvenIfLimited(String localeString, String path, boolean isError, boolean isMissing) {
 
-        // don't limit new locales or errors
+        // Allow errors to be fixed
+        if (isError) {
+            return true;
+        }
 
-        if (SubmissionLocales.ALLOW_ALL_PATHS.contains(localeString) || isError) {
+        // for new locales, allow basic paths
+        if (SubmissionLocales.ALLOW_ALL_PATHS_BASIC.contains(localeString) &&
+            // Only check coverage level for these locales
+            isPathBasicOrLess(localeString, path)) {
             return true;
-        } else {
-            int debug = 0; // for debugging
         }
 
         // all but specific locales are otherwise locked
-
         if (!SubmissionLocales.LOCALES_ALLOWED_IN_LIMITED.contains(localeString)) {
             return false;
-        } else {
-            int debug = 0; // for debugging
         }
 
-        // in those locales, lock all paths except missing and special
-
-        if (isMissing) {
+        // in TC Org locales, lock all paths except missing and special
+        if (isMissing && TC_ORG_LOCALES.contains(localeString)) {
             return true;
-        } else {
-            int debug = 0; // for debugging
         }
 
         if (pathAllowedInLimitedSubmission(path)) {
             return true;
-        } else {
-            int debug = 0; // for debugging
         }
 
         return false; // skip
     }
 
+    private static boolean isPathBasicOrLess(String localeString, String path) {
+        return SubmissionLocalesCache.getCoverageLevel(localeString, path).compareTo(Level.BASIC) <= 0;
```

Comment:
Question: I thought that getting the coverage level was already cached; if that is the case then we have duplicate caching here. (But I'm not sure.)

Reply:
It's cached elsewhere but not in getinstance. This code doesn't have access to the survey tool cache.